src/api/routes.ts

```diff
@@ -86,6 +86,8 @@ export const buildGetMembersRoute = (ids: UUID[]) =>
   `${MEMBERS_ROUTE}?${qs.stringify({ id: ids }, { arrayFormat: 'repeat' })}`;
 export const buildPatchMember = (id: UUID) => `${MEMBERS_ROUTE}/${id}`;
 export const buildDeleteMemberRoute = (id: UUID) => `${MEMBERS_ROUTE}/${id}`;
+export const buildUpdateMemberPassword = () =>
```

Comment:
I see we're not always following the convention, but ideally this should be have route at the end!
```suggestion
export const buildUpdateMemberPasswordRoute = () =>
```